plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-products-controller.php

```diff
@@ -987,6 +992,11 @@ public function get_item_schema() {
 					'context'     => array( 'view', 'edit' ),
 				),
 				'sku'                   => array(
+					'description' => __( 'SKU.', 'woocommerce' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'unique_id'             => array(
```

Comment:
Hmm should the `description` here for `unique_id` be: `'GTIN, UPC, EAN or ISBN'` ?
And we keep `'Unique identifier.'` for the `sku` field?

Reply:
Just for clarification: `sku` is unique at the store level vs. the new field shall hold a globally unique identifier such as `GTIN`.
Hence, the proposed new `unique_id` would also be unique at the store level.
Maybe calling it `global_unique_id` could be more explicit?

Reply:
I like it. I will rename it